demo/client/src/ml-models/deploy-model.ts

```diff
@@ -332,7 +331,13 @@ export class ModelDeployer {
 		if (options.saveTransactionHash === undefined) {
 			options.saveTransactionHash = (() => { })
 		}
-
+		if (options.initialChunkSize === undefined) {
+			options.initialChunkSize = 200
```

Comment:
The defaults should not be the same for all models because the "chunks" have different byte sizes depending on the model.

Reply:
can you pls explain a bit, having some doubt over this

Reply:
The default value should not be set in this method.

Reply:
So where should I set it, can you pls guide me

Reply:
Most of the methods already sets up a hard-coded defaults. We need an easy way to override all of those and keep the current values as the default for that method.

Reply:
kindly check now